packages/hoc-input/src/styledComponents/Wrapper.js

```diff
@@ -61,6 +59,10 @@ const Wrapper = styled.div`
     }
 
     input {
+      height: ${ifSmall("30", "46")}px;
```

Comment:
is it 30 instead of 32 because of the 1px borders? would it be better to use the multiple of 8 values and box-sizing border box?

Reply:
It won't work because we have `border: none` on `input`